Makefile

```diff
@@ -1,3 +1,17 @@
+prefix=/usr/local
+datarootdir=$(prefix)/share
+
+SCRIPT_FILES=gim-commit
```

Comment:
SCRIPT stands for "the script"? I mean, ... "the script files"? Isn't GIM more appropriate than SCRIPT here?

 - GIM_FILES
 - GIM_SCRIPTS